README.md

```diff
@@ -75,6 +75,24 @@ This example results in both a unified CSV along with all the corresponding CRDs
 
 Run `make ocs-registry` to generate the registry bundle container image.
 
+## Prerequisites
+
++ OCS Operator will install its components only on nodes marked for OCS (i.e. nodes with label `cluster.ocs.openshift.io/openshift-storage=''`)
+
+  To label the nodes from CLI,
+
+  ```console
+  $ oc label nodes <NodeNames> cluster.ocs.openshift.io/openshift-storage=''
+  ```
+
++ Nodes marked for OCS usage will be tainted (with taint `node.ocs.openshift.io/storage=true:NoSchedule`) and pods that do not tolerate it will be unscheduleable on these nodes.
+
+  To taint the nodes from CLI,
+
+  ```console
+  $ oc adm taint nodes <NodeNames> node.ocs.openshift.io/storage=true:NoSchedule
+  ```
```

Comment:
Can you add a note about using `PreferNoSchedule`.  When deploying from the CLI, a user would first set the taints and then create the deployment. If we taint wprker nodes as NoSchedule right away, then nothing gets scheduled on there and nothing get deployed. Tainting a node noschedule is okay from the console, as the taint is done when the storage cluster is created. This happens after the deployment is done, and any scheduled pods should continue running after the taints are applied.

Reply:
This is the issue whether you use UI/CLI. PreferNoSchedule is same as not having a taint. So, we'd rather not ask user to taint . But, then there will be 2 different workflows which we need to avoid.

So, while I am working on finding a permanent fix for this, we need to suggest everyone not to taint nodes but keep this doc as it is because this is the ultimate truth that Placement Config is introducing.

Reply:
Tainting just before StorageCluster creation is working. So move this section to just before the instructions on creating StorageCluster.

Reply:
Done.